src/pages/Home.js

```diff
@@ -11,12 +12,16 @@ const Hero = styled.div`
   text-align: center;
 `;
 
+const Query = query => {
+  console.log(query);
+};
 const Home = () => {
   return (
     <main>
       <Hero>
         <Wrapper>
           <h1>Home Page</h1>
+          <SearchBox onSubmit={Query} />
```

Comment:
You don't really need a new function here, adding `onSubmit={console.log}` should be enough.